agent/consul/acl_endpoint.go

```diff
@@ -26,6 +26,16 @@ const (
 // Regex for matching
 var validPolicyName = regexp.MustCompile(`^[A-Za-z0-9\-_]{1,128}$`)
 
+const (
+	// maxTokenExpirationDuration is the maximum difference allowed between
+	// ACLToken CreateTime and ExpirationTime values.
+	maxTokenExpirationDuration = 24 * time.Hour
```

Comment:
Presumably this is not mandatory though for all tokens? So the max is really infinite but this is the max lifetime, assuming there _is_ a finite lifetime?

Reply:
I actually did set an upper bound here. I can remove it if you think an upper bound doesn't seem necessary in practice

Reply:
I meant, this upper bound doesn't affect _all_ tokens right (i.e. expiring is not mandatory)? Otherwise all your bootstrap admin tokens will eventually expire and you'll be locked out!

Given that, the comment might make that more clear.

Reply:
Clarified.